ts/features/mvl/types/mvlData.ts

```diff
@@ -26,11 +25,11 @@ export type MvlId = string & IUnitTag<"MvlId">;
 export type MvlAttachment = {
   id: MvlAttachmentId;
   // a display name for the file
-  name: string;
-  // atm we have to distinguish only the pdf files from the others for a custom (future) view
-  contentType: Extract<ContentType, "application/pdf"> | "other";
+  displayName: string;
+  // a generic content type for a file
+  contentType: string;
```

Comment:
Shouldn't this be a [ContentType](https://github.com/pagopa/io-app/pull/3618/files#diff-5c0d6c0e281d02e1bfa1cd3a3b96adf59befd5faf23acb1f14d63e022d48e994R16)?

Reply:
As reported in the pr list of changes:
- changed `contentType` to `string` in order to support generic content type

This was a necessary change in order to consider all content types without excluding the unmapped ones.